test/react-native/batch-integration.tsx

```diff
@@ -51,7 +51,10 @@ describe('React Native', () => {
     return action.type === 'APPEND' ? prev + action.body : prev
   }
 
-  afterEach(() => rtl.cleanup())
```

Comment:
This is automatically handled by the auto-cleanup from `@testing-library/react-native`. We just need to make sure we restore the original timers now.